lib/chef-dk/skeletons/code_generator/recipes/custom_resource.rb

```diff
@@ -0,0 +1,13 @@
+
+context = ChefDK::Generator.context
+cookbook_dir = File.join(context.cookbook_root, context.cookbook_name)
+
+resource_dir = File.join(cookbook_dir, "resources")
+resource_path = File.join(resource_dir, "#{context.new_file_basename}.rb")
+
+directory resource_dir
+
+template resource_path do
+  source "resource.rb.erb"
```

Comment:
It totally is blank. I was thinking that I would get this command pushed through and then follow this pull request up with another that uses a new template with the boilerplate.